crates/miden-lib/build.rs

```diff
@@ -87,6 +88,8 @@ fn main() -> Result<()> {
     let mut assembler =
         compile_tx_kernel(&source_dir.join(ASM_TX_KERNEL_DIR), &target_dir.join("kernels"))?;
 
+    compile_executor(&source_dir, &target_dir, &assembler)?;
```

Comment:
Related to the previous comment, I'd probably run this function as a part of `compile_tx_kernel()` and name it accordingly.